docs/Makefile

```diff
@@ -4,7 +4,7 @@
 export PYVISTA_OFF_SCREEN := True
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -j auto
```

Comment:
If I understand correctly this triggers multicore builds. This might explain (at least part of) the CPU and memory load I mentioned in a previous comment. I wonder if something's off about the parallel workers (their number?) that leads to that behaviour. Perhaps it's all working as expected but the temporal distribution feels unusual thanks to the multiple cores.

Reply:
I have no idea what magic they use behind their scenes, but it's aprox 2-3 times faster locally building the docs from scratch (minus gallery) with multiple cores enabled on my 4 core machine.

Reply:
I guess the additional memory need is due to each process needing its own share, effectively multiplying required memory. It might all make sense, I'll just have to be a bit more careful when building the docs.

Reply:
One issue with leaving this as "auto" is it will be greedy and use as many CPUs as possible.  I'll instead make it default to 2.  This way we'll have faster builds on our CI, but we won't crash locally.  We should get a 2x ish speedup for the API generation.